KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -81,6 +81,10 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
         footerView.translatesAutoresizingMaskIntoConstraints = false
         footerView.accessibilityIdentifier = "footer_view"
         footerView.backgroundColor = .white
+        footerView.layer.shadowColor = KarhooUI.colors.textLabel.cgColor
+        footerView.layer.shadowOpacity = 1
+        footerView.layer.shadowOffset = CGSize(width:0,height:-1)
+        footerView.layer.shadowRadius = 0.5
```

Comment:
I'm quite sure we should have a shadow radius standardized within `UIConstants`. 

(`UIConstants.Alpha.shadow`)

Reply:
`Alpha` is concerned with opacity, not radius. We have `CornerRadius`, but I'm not sure if we should add a `ShadowRadius` as well because I'm not conviced we'll be using it in more than one place. Up for discussion.

Reply:
@mionescu95 could you please add a `ShadowRadius` struct in `UIConstants` right below `CornerRadius`? And add this 0.5 value as `border`? I'm not comfortable with reusing alpha to describe the radius. We may want to tweak the value of one or the other in the future which will bring unwanted consequences.
cc: @AleksanderWedrychowskiKarhoo